python/tank/bootstrap/configuration.py

```diff
@@ -25,11 +25,12 @@ class Configuration(object):
 
     (LOCAL_CFG_UP_TO_DATE, LOCAL_CFG_MISSING, LOCAL_CFG_DIFFERENT, LOCAL_CFG_INVALID) = range(4)
 
-    def __init__(self, path):
+    def __init__(self, path, descriptor=None):
```

Comment:
That shouldn't be defaulted. It's leftover from an earlier implementation. Thanks for catching it!